components/atoms/typography.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
```

Comment:
It feels like that these files are duplicated. Are you generating these somehow ?

If I have files that purely have styled components i like to differentiate them with a naming convention.
I would usually do something like typography.styles.tsx - that way for me it is easier to search for a given file.